src/components/admin/partners/helpers/mediaAboutUsData.ts

```diff
@@ -32,14 +32,6 @@ export const articles: ArticleData[] = [
       '“Нова дарителска платформа в полза на нуждаещите се и организациите, които им помагат, заработи от началото на август. Нейното име и адрес е podkrepi.bg...”',
     url: 'https://bnr.bg/post/101689024/novata-daritelska-platforma-podkrepibg-veche-pomaga',
   },
-  {
-    img: '/img/partners/mediaAboutUs/darik.svg',
-    title: 'Darik Radio',
-    subtitle: 'Нова дигитална платформа стимулира дарителската култура в България',
-    description:
-      '“От началото на месец август стартира нова българска дарителска платформа, подкрепяща нуждаещите се и организациите, които им помагат. Тя се нарича Подкрепи.бг и...”',
-    url: 'https://darik.bg/darik.bg/nova-digitalna-platforma-stimulira-daritelskata-kultura-v-balgaria',
```

Comment:
Article's link is available. No need to remove it.
[https://darik.bg/nova-digitalna-platforma-stimulira-daritelskata-kultura-v-balgaria](https://darik.bg/nova-digitalna-platforma-stimulira-daritelskata-kultura-v-balgaria)

Reply:
The problem is that the link is incorrect. I will update it later. Thank you!